app/components/home/Home.js

```diff
@@ -128,11 +133,21 @@ export default () => (
       <div className="section">
         <div className="section-content">
           <div className="img-wrapper">
-            <img src={require('../../../src/images/work.jpg')} id="work"></img>
+            <img
+              src={require('../../../src/images/sunsetjump.png')}
```

Comment:
I like an image on mobile. I find the gif strange on mobile b/c I can only see the woman's hands/feet clipping the frame every so often